cft/misc/s3-bucket/test.tf

```diff
@@ -1,13 +1,17 @@
 resource "aws_s3_bucket" "mybucket" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure versioning is enabled for AWS S3 Buckets**
 
**Vulnerability**
Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites.
 
**Remediation**
You can use versioning to preserve, retrieve, and restore every version of every object stored in your Amazon S3 bucket. With versioning, you can easily recover from both unintended user actions and application failures. When you enable versioning for a bucket, if Amazon S3 receives multiple write requests for the same object simultaneously, it stores all of the objects.

source: accurics-